internal/compiler-bridge-test/src/test/scala/sbt/internal/inc/ExtractUsedNamesPerformanceSpecification.scala

```diff
@@ -23,8 +26,22 @@ class ExtractUsedNamesPerformanceSpecification extends CompilingSpecification {
   }
 
   val TestResource = "/ExtractUsedNamesPerformance.scala.source"
-  // Some difference between 2.10, 2.11, and 2.12
-  val scalaDiff = Set("Any", "Nothing", "_root_", "StringAdd", "Option")
+  // Some difference between 2.10, 2.11, 2.12, and 2.13
+  val scalaDiff = Set(
+    "Any",
+    "Nothing",
+    "_root_",
+    "StringAdd",
+    "Option",
+    "?0",
+    "package",
+    "acme",
```

Comment:
Why is `acme` and `ModuleSerializationProxy` an exception for Scala 2.13?

Reply:
acme was missing from the list of extracted names, and ModuleSerializationProxy showed up as an extracted name. We can open issue for these.